s3/pkg/meta/db/drivers/tidb/bucket.go

```diff
@@ -483,6 +483,12 @@ func (t *TidbClient) DeleteBucket(ctx context.Context, bucket *Bucket) error {
 	_, err := t.Client.Exec(sqltext, bucket.Name)
 	if err != nil {
 		return handleDBError(err)
+	} else {
+		sqltext := "delete from bucket_versionopts where bucketname=?;"
```

Comment:
Suggest to use transaction here, make sure buckets and bucket_versionopts both success, otherwise roll back.

Reply:
yes did the code change, will push new commit

Reply:
This is not transaction, see DeleteObject in tidb/object.go please.

Reply:
@nguptaopensds , this comment is handled?